idl/riscv-plic.idol

```diff
@@ -0,0 +1,35 @@
+Interface(
```

Comment:
I think we should write some documentation regarding the use of PLIC driver by any task. There is certain expectation w.r.t. enable/disable and complete operations. It would be good to document this to make it easier for task developers.

Reply:
Agreed. I plan on writing that once the API is approved.